### PR TITLE
Turtle evaluation tests

### PR DESCRIPTION
**Pull request by afs**
Includes

- Turtle evaluation tests, include annotation tests
- N-Triples* syntax tests
- Every NT* syntax test is copied, and in the NT* manifest as a Turtle syntax test

### CODE REVIEW

File: tests/turtle/eval/manifest.ttl [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
+
+PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
+PREFIX rdft:   <http://www.w3.org/ns/rdftest#>
+PREFIX :       <#>
+
+<>  rdf:type mf:Manifest ;
+    rdfs:label "Turtle* EvaluationSyntax" ;
```
Comment (gkellogg) [inline tests/turtle/eval/manifest.ttl:13]: ```suggestion`
    rdfs:label "Turtle* Evaluation Tests" ;
```

**Review by gkellogg — CHANGES_REQUESTED** [review]
Some of the NTriples test files referenced from the manifest differ from the actual test files:

* `ntriples-star-syntax-bad-*.nt` vs `ntriples-star-bad-syntax-*.nt`
* `nt-ttl-star-syntax-bad-*.nt` vs `nt-ttl-star-bad-syntax-*.nt`

Also, please update /index.html with references to these test manifests; HTML version, if it's generated.

Otherwise, I pass all the tests.

File: tests/turtle/eval/manifest.ttl (hunk 1 above)
Comment (afs) [reply tests/turtle/eval/manifest.ttl]: Changes made.  Thanks for catching the errors.

**Comment by afs** [comment]
> Also, please update /index.html with references to these test manifests; HTML version, if it's generated.

I have tried to do that. Added to this PR.

@pchampin - I had to change the `make-html-manifests.py` to remove the validation of the id name because it was checking for `#turtle-star-`.

**Comment by pchampin** [comment]
> @pchampin - I had to change the make-html-manifests.py to remove the validation of the id name because it was checking for #turtle-star-.

No problem... I initially added that to detect copy-paste errors (copying a test and forgetting to change its name or its id) but I should probably run this check in a separate script...

**Review by gkellogg — APPROVED** [review]
Thanks, but still no link to the Turtle tests from /index.html. Otherwise, LGTM.

**Comment by afs** [comment]
HTML rebuilt now; links on home page (and fixed the insecure links warning).

**Comment by pchampin** [comment]
Thanks a lot :)
One thing I believe is missing: a test case for Turtle* and N-Triples* where there is no space between `<<` and the embedded subject. I think I encountered some implementations that didn't support it, but my understanding of the grammar is that it should (even though that's not very readable...). Do you agree?

**Comment by afs** [comment]
It is possible.

`<<<uri> :q 123>>` and `<<<<:s :p :o>> :q :z>>` and similarly the trailing case.

I'll add some tests for this.

**Pull request merged**